indra/llui/lllineeditor.cpp

```diff
@@ -1590,19 +1595,28 @@ BOOL LLLineEditor::handleKeyHere(KEY key, MASK mask )
 				deselect();
 			}
 
-			BOOL need_to_rollback = FALSE;
+			bool need_to_rollback = false;
+			bool prevalidator_failed = false;
 
 			// If read-only, don't allow changes
-			need_to_rollback |= (mReadOnly && (mText.getString() == rollback.getText()));
+			need_to_rollback |= (mReadOnly == TRUE) && (mText.getString() == rollback.getText());
```

Comment:
But why?
Please use lowercase `bool` if it's required.

Reply:
> But why?
> Please use lowercase `bool` if it's required.

Just trying to make Visual Assistant happy
![image](https://github.com/secondlife/viewer/assets/124201357/ab500781-e172-4dd7-8e72-89bcbca09aec)

Reply:
It will get fixed by itself after maint A merge.

Reply:
Fixed
![image](https://github.com/secondlife/viewer/assets/124201357/ba3fd96e-d4dd-426d-8c6d-f1339f571478)